spec/xclarity_client_discovery_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe XClarityClient do
+
+  before :all do
+    WebMock.allow_net_connect! #-- Uncomment this line if you're testing with a external mock.
+
+    conf = XClarityClient::Configuration.new(
+    :username => ENV['LXCA_USERNAME'],
+    :password => ENV['LXCA_PASSWORD'],
+    :host     => ENV['LXCA_HOST'],
+    :port     => ENV['LXCA_PORT'],
+    :auth_type => ENV['LXCA_AUTH_TYPE'],
+    :verify_ssl => ENV['LXCA_VERIFY_SSL']
+    )
+
+    @client = XClarityClient::Client.new(conf)
+
+    @host = ENV['LXCA_HOST']
+  end
+
+  before :each do
```

Comment:
Just a question, why you added these lines?

```
before :each do
     @includeAttributes = %w(lastUpdateElapsedTime discoveryInProgress)
     @excludeAttributes = %w(lastUpdateElapsedTime discoveryInProgress)
  end
```
I think that isn't being used.

Reply:
Thanks for catching this. Addressed now.